code/src/components/TimeStamp.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import moment from 'moment'
+
+
+const TimeStamp = () => {
+
+return (
+
+    <div>
+            {moment().calendar()}
+    </div>
```

Comment:
This is the part I was mentioning in the beginning, that the date time updates every single todo item instead of saving it to the todo. I haven't figured out how to solve this (since I was struggling with this too) but I will look at it and get back to you if I figure it out. I'm thinking it should not be so hard.